src/gcp_scanner/credsdb.py

```diff
@@ -189,12 +189,12 @@ def find_creds(explicit_path: Optional[str] = None) -> List[str]:
         search_paths.append(full_path)
 
   for dir_path in search_paths:
-    print(f"Scanning {dir_path} for credentials.db")
+    logging.info(f"Scanning {dir_path} for credentials.db")
```

Comment:
For log calls, don't use f-strings, instead do `logging.info("Scanning %s for credentials.db", dir_path)`